packages/components/src/components/primitives/CircularProgress/CircularProgress.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { CircularProgress } from './CircularProgress';
+import { HStack, Text, VStack } from '@chakra-ui/react';
+
+export default {
+  component: CircularProgress,
+  title: 'Components/Primitives/CircularProgress',
+  argTypes: {
+    size: {
+      control: { type: 'number' },
+    },
```

Comment:
this is defined as a `number` in here but the value in the [arguments](https://github.com/cmpsr/composer/pull/306/files#diff-59fcda40313833c8ba0b5580caa163e1e454256230dfc80cacb53ff2d6094636R44) is a string.

Also the behaviour is errant:
<img width="1583" alt="Screenshot 2022-01-18 at 18 38 33" src="https://user-images.githubusercontent.com/884028/149989767-7c1dd436-fc1d-40a0-afaf-571ba596f01e.png">
<img width="1587" alt="Screenshot 2022-01-18 at 18 38 42" src="https://user-images.githubusercontent.com/884028/149989772-c86c0c54-955e-4f49-adb5-eca3a7390cfb.png">

Reply:
done!